src/components/Tag.js

```diff
@@ -54,6 +54,7 @@ Tag.propTypes = {
   tag: PropTypes.shape({
     id: PropTypes.string.isRequired,
     className: PropTypes.string,
+    key: PropTypes.string,
```

Comment:
this is already optional, omitting it will just default to (something like) the old behavior